app/javascript/member-facing/cookieConsent.js

```diff
@@ -0,0 +1,31 @@
+import { formatMessage } from '../util/TranslationsLoader';
+
+const initializeCookieConsent = () => {
+  $(() => {
+    if (isEEA()) {
+      const locale = window.champaign.page.language_code || 'en';
+      window.cookieconsent.initialise({
+        theme: 'block',
```

Comment:
Is there a reason why we're not simply passing the country code to `cookieconsent.initialise`? 

If we pass `law.countryCode`, it will follow some country specific rules (can user revoke, can user change their mind, etc): https://github.com/insites/cookieconsent/blob/master/src/cookieconsent.js#L1374-L1387